src/pages/account/profile/addresses/[id]/update.tsx

```diff
@@ -0,0 +1,83 @@
+import {
```

Comment:
This page should be called `src/pages/account/profile/addresses/[id]/edit.tsx ` instead of update, because at this point, it's not the update yet, it's only the edit part of the update flow

Reply:
Still following the RESTful routes, the update form is on the `resources/:id/edit` page.
On this page, the user doesn't really updates its profile, s·he just "edit" the data before the form makes the call to update it.
"It may be a detail for you, but for me it's veut dire beaucoup" 🤣

Reply:
Well, I see your point, but isn't this just bringing unnecessary confusion between `edit` and `update`?

Reply:
As for above, I think @Fenntasy can give us its advice.
Coming from Rails I still have the habit of this naming convention for app which aren't SPA and may lack of objectivity on this

Reply:
> who dares summon me?
> - a confused demon

> bringing unnecessary confusion between edit and update

just because I'm confused there, do we have both at the moment? 

Other than that, in my mind, the web page would be `edit` and the api page would be `update`. And I think of it from the point of view of the user:
- I want to edit my personal data (the web page)
- I want to send my data to something that updates my personal data.

For me the edit is the writing part. 

Does it answer your questions or bring more? 😅

Reply:
I understand the difference of context between `edit` and `update`. My point was,  is it a necessary to make the distinction here. But I'll change it.

Reply:
> My point was, is it a necessary to make the distinction here. 

Could you elaborate? Do you think it's simpler to have the same naming on both sides or is that needing to have two verbs or something else?

Reply:
My point is, even if the user action is to edit his profile, in the end, all he wants it to update his profile. I don't see why we should split the naming for the flow 🤔 .

Reply:
> "It may be a detail for you, but for me it's veut dire beaucoup"

I'll make the changes then.